docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -571,6 +571,21 @@ the state of the table to a previous snapshot id::
 
     CALL iceberg.system.rollback_to_snapshot('testdb', 'customer_orders', 8954597067493422955)
 
+Register table
+----------------
+Iceberg supports creating table with existing metadata and data files.
```

Comment:
`creating` is not appropriate.

The procedure _registers_ into the metastore an Iceberg table.

Reply:
Modified the text. Please review.